src/components/util/AdjustableColumns/AdjustableColumns.js

```diff
@@ -77,7 +77,8 @@ export default class AdjustableColumns extends Component {
       <Col lg={lg} hidden={lg === 0} key={i}>
         <Row>
           <Col lg={12} className={'text-center'} style={{
-            marginBottom: '-0.5em',
+            marginTop: '0.25em',
+            marginBottom: '0.25em',
```

Comment:
The original was intentional, to conserve space, but I agree it looks better (and more apparent) like this.